Update Dockerfile-php8.1-pgsql13

pdtouch opened the pull request:
Changed Scripts configuration to 
 1. Include hostname in the URL for accessing the docker-tripal site remotely and
 2. Add 'verbose' option in loggin to see messages on the Drupal screen - instead of having to access them from error logs

<!--- Thank you for contributing! -->
<!--- Provide a general summary of your changes in the Title above -->
<!--- See our Contribution Guidelines here:
          https://github.com/tripal/tripal/blob/7.x-3.x/CONTRIBUTING.md -->


<!---  Please set the header below based on the PR type:
# New Feature
# Bug Fix
# Tripal 4 Core Dev Task  --->

#

Issue #

<!--- Enter the Tripal version this PR applies to (i.e. either 3 or 4 ;-p) --->
### Tripal Version: 

## Description
<!--- Describe your changes in detail -->
<!--- Why is this change required? What problem does it solve? -->

## Testing?
<!--- Please describe in detail how to test these changes. -->
<!--- Reviewers will use this section to test the submission! -->
<!--- If you've implemented PHPUnit tests, you can describe the test cases here. -->

pdtouch committed:
Update Dockerfile-php8.1-pgsql13

Changed Scripts configuration to 
 1. Include hostname in the URL for accessing the docker-tripal site remotely and
 2. Add 'verbose' option in loggin to see messages on the Drupal screen - instead of having to access them from error logs

dsenalik commented:
A similar changed as performed in item 1 is documented at https://tripaldoc.readthedocs.io/en/latest/install/docker.html#the-provided-host-name-is-not-valid-for-this-server where a hostname is explicitly specified.

pdtouch committed:
Update Dockerfile-php8.1-pgsql13

Added missing quote after closing square bracket

pdtouch committed:
Update Dockerfile-php8-pgsql13

Changes to Scripts to add verbose debug and add server-name to URL for remote access.

laceysanderson commented:
Note: the automated testing is not running on this PR since this is @pdtouch first PR. However, they do run on his fork. You can see the automated tests pass here:
- PHP 8.0 https://github.com/pdtouch/tripal/actions/runs/4659700047
- PHP 8.1 https://github.com/pdtouch/tripal/actions/runs/4659648326

PS. If anyone knows how to enable these tests to run in our repo that would be great. Apparently the github docs are out of date on how to do this: https://docs.github.com/en/actions/managing-workflow-runs/approving-workflow-runs-from-public-forks

dsenalik commented:
@laceysanderson good to know. There was a button that I clicked to authorize running the checks, but I guess that wasn't enough.

laceysanderson reviewed (approved):
Thank you for the PR @pdtouch!

✅ Code review: all looks good on my end 👍 
✅ Automated testing: the automated testing passes in your fork showing that the dockerfile builds perfectly with these changes and that the site is still functional with them.

Thus this is ready to be merged! No manual testing needed :-)

pdtouch commented:
Thank you.

laceysanderson commented:
❤️ Congrats -now your name shows as an official contributor to Tripal core 🎉  I look forward to your future PRs ❤️

pdtouch commented:
Feel honored to contribute to this amazing project and be a part of this awesome group.
